src/mouse_events.cpp

```diff
@@ -292,6 +293,10 @@ void mouse_handler::mouse_motion(int x, int y, const bool browse, bool update, m
 	if ( (!selected_hex_.valid()) && un && current_paths_.destinations.empty() &&
 			!gui().fogged(un->get_location()))
 	{
+		//   Only process unit if toggler not preventing unit
+		// processing.
+		if (!preventing_units_highlight_) {
+
```

Comment:
Please add a commit that does nothing but fix the indentation in this block.

Reply:
Will comply request in the end.